content/articles/2018-10-devtools-2.0.0.Rmarkdown

```diff
@@ -0,0 +1,256 @@
+---
+title: devtools 2.0.0
+date: 2018-10-26
+slug: devtools-2-0-0
+author: Jim Hester
+categories: [package]
+description: >
+    Make package development easier by providing R functions that simplify and expedite common tasks.
+photo:
+  url: https://unsplash.com/photos/wEJK4q_YlNQ
+  author: Hunter Haley
+---
+
+## Introduction
+
+[devtools] 2.0.0 is now on CRAN!
+
+devtools makes package development easier by providing R functions that
+simplify and expedite common tasks. [R Packages] is a book based around this
+workflow.
+
+devtools 2.0.0 is a major release that contains work from the past year and a
+half, since the major devtools release (1.13.0).
+
+## Breaking changes
+
+There are a handful of breaking changes in this release, mainly done in the
+interest of simplifying and standardizing the function APIs.
+
+The changes should affect relatively little user code, but have required
+developers to make changes. Common errors and ways to work around them
+as well as the full list of changes can be found in the [Breaking
+changes](https://github.com/r-lib/devtools/blob/master/NEWS.md#breaking-changes)
+section of devtools'
+[NEWS](https://github.com/r-lib/devtools/blob/master/NEWS.md). If you
+discover something missing, please let us know so we can add it.
```

Comment:
Should these link, instead, to the NEWS in the pkgdown site?

http://devtools.r-lib.org/news/index.html#breaking-changes